src/Me/components/Select/__snapshots__/Select.test.js.snap

```diff
@@ -25,7 +25,7 @@ exports[`Select component renders properly 1`] = `
           <input
             aria-autocomplete="list"
             autocapitalize="none"
-            autocomplete="off"
+            autocomplete="nope"
```

Comment:
Never tested it but we're 2020, why we still have to hack browsers? 😡

Reply:
![image](https://user-images.githubusercontent.com/16106431/102698431-06388780-4235-11eb-8045-995263cf8e4a.png)